build/vega-lite-schema.json

```diff
@@ -1032,32 +1032,47 @@
       "$ref": "#/definitions/CompositeUnitSpecAlias",
       "description": "Unit spec that can have a composite mark."
     },
-    "Condition<(string|number)>": {
-      "additionalProperties": false,
+    "Condition<(LegendFieldDef|ValueDef<number>)>": {
       "properties": {
         "selection": {
           "$ref": "#/definitions/LogicalOperand"
-        },
-        "value": {
-          "type": [
-            "string",
-            "number"
-          ]
         }
       },
       "required": [
-        "selection",
-        "value"
+        "selection"
+      ],
+      "type": "object"
+    },
+    "Condition<(LegendFieldDef|ValueDef<string>)>": {
```

Comment:
@domoritz  looks like we have a bug in the schema generator?

Some properties are missing here